### ISSUE TITLE
The docs should have an example of how to create/use custom contracts

### ISSUE THREAD

**Issue by tibetsprague** [daostack/arc#370]
In general we need to have more step by step walk throughs of how to use Arc and Arc.js, and how to build on top of them, especially creating new voting machines, schemes and global constraints and deploying and using them.

**Comment by dev-matan-tsuberi** [comment]
Something like [this](https://daostack.github.io/arc/contracts/globalConstraints/README/) I guess but more detailed and comprehensive?

**Comment by dev-matan-tsuberi** [comment]
Also, I think, it may be better to unify Arc & Arc.js docs in the `docs` repo and explain things in an "end-to-end" approach using the whole stack, instead of a maybe more modular but less clear way of splitting docs in each repo.

What do you think?

**Comment by tibetsprague** [comment]
I agree that it would be nice to unify the docs, or at least have a guides website that links to both sets of docs and has guides that includes examples for developers that include working in both Arc and Arc.js

**Comment by tibetsprague** [comment]
@dev-matan-tsuberi the example you include above is cool for GlobalConstraints. I think a new VotingMachine example would be more complicated and need to be thorough

**Comment by orenyodfat** [comment]
@dev-matan-tsuberi , do you feel like doing that ?

**Comment by dev-matan-tsuberi** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [The docs should have an example of how to create/use custom contracts](https://trello.com/c/vIirdto0/148-the-docs-should-have-an-example-of-how-to-create-use-custom-contracts)
